Decisions.TruBot/Data/TruBotConstants.cs

```diff
@@ -0,0 +1,9 @@
+namespace Decisions.TruBot.Data
+{
+    public class TruBotConstants
+    {
+        public const string TRUBOT_ROOT_FOLDER_ID = "01HNB8V81CT9RSWA6ZB6S9ZRT0";
```

Comment:
We should touch base with Jason and Jinesh. 
This is correct based on my suggestion, but I was just asked to change the HL7 folder I used for reference, to be a project specific folder.

Reply:
This was also in the requirements doc:

- Once the module is installed, this folder will live under the System folder and only be available for System Administrators
- Location will be System > Jobs and Events